pyproject.toml

```diff
@@ -75,16 +75,22 @@ marimo = "marimo._cli.cli:main"
 homepage = "https://github.com/marimo-team/marimo"
 
 [project.optional-dependencies]
-sql = ["duckdb >= 1.0.0", "polars[pyarrow] >= 1.9.0"]
+sql = [
+    "duckdb>=1.1.0",
```

Comment:
nit: If `duckdb 1.0` is still compatible with this change, we should prefer it over `1.1` so not to break users who haven't upgraded.